models/list_option/list_option.go

```diff
@@ -56,12 +57,12 @@ func FindByAction(action string) *ListOptionStruct {
 	err := table.QueryRow("select id from list_options where action = ?;", action).Scan(&listOptionID)
 	if err != nil {
 		logging.SharedInstance().MethodInfo("ListOption", "FindByAction").Info("cannot find list_option")
-		return nil
+		return nil, err
 	}
-	return NewListOption(listOptionID, action)
+	return NewListOption(listOptionID, action), nil
 }
 
-func FindByID(id sql.NullInt64) *ListOptionStruct {
+func FindByID(id sql.NullInt64) (*ListOptionStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindByID should have comment or be unexported